docs/user/aws/customization.md

```diff
@@ -7,6 +7,7 @@ Beyond the [platform-agnostic `install-config.yaml` properties](../customization
 * `amiID` (optional string): The AMI that should be used to boot machines for the cluster.
     If set, the AMI should belong to the same region as the cluster.
 * `region` (required string): The AWS region where the cluster will be created.
+* `publicZoneID` (optional string): The public zone ID of the base domain DNS zone.
```

Comment:
can you include an example of the install-config.yaml like others below.

Reply:
done!